docs/semgrep-cloud-platform/jira.md

```diff
@@ -0,0 +1,91 @@
+---
+slug: jira 
+append_help_link: true
+title: Jira
+hide_title: true
+description: "Send Semgrep findings to your Jira project."
+tags:
+    - Semgrep Cloud Platform
+    - Team & Enterprise Tier
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+<ul id="tag__badge-list">
+{
+Object.entries(frontMatter).filter(
+    frontmatter => frontmatter[0] === 'tags')[0].pop().map(
+    (value) => <li class='tag__badge-item'>{value}</li> )
+}
+</ul>
+
+# Creating Jira Tickets
+
+<!-- It's Jira, not JIRA :) -->
+
+:::tip Try our Jira integration (Beta)
+Existing customers can try our Jira integration for free by contacting their account manager.
+:::
+
+The Semgrep Jira integration allows you to create Jira tickets based on your Semgrep Code or Supply Chain findings.
+
+## Enabling your Jira integration
+
+:::info Prerequisites
+* Your Jira user must be able to create a Jira API key with admin level privileges.
+* Jira can only be enabled through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up your Jira integration.
+* To create tickets, you must [add or onboard the project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
+:::
+
+To integrate Jira, follow these steps:
+
+1. In [Semgrep Cloud Platform](https://semgrep.dev/login), go to **Settings** > **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)**.
+2. On the **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)** page, click **Add Integration** (or **Setup First Integration** if this is your first integration), and then select **Jira**.
+3. Follow the on-screen instructions to add your Jira API key.
+
+## Configuring your default project and ticket type
+
+Use the drop-downs to select the default project and ticket type. These settings can be changed later from the integrations page.
+
+![Jira configuration modal](/img/jira-configure-defaults.png) 
+
+**Figure** A sample Jira integration
+
+## Creating tickets
+
+### Code
+
+:::note
+Creating tickets for many findings at once may take some time. Tickets that take longers than 10 seconds to create will be shown in the UI once the page is refreshed.
```

Comment:
(Not a suggestion - just a note - the "will" here totally makes sense and doesn't have to be changed.)